telnyx_rtc/src/main/java/com/telnyx/webrtc/sdk/TelnyxClient.kt

```diff
@@ -1414,8 +1425,12 @@ class TelnyxClient(
                     object : PeerConnectionObserver() {
                         override fun onIceCandidate(p0: IceCandidate?) {
                             super.onIceCandidate(p0)
-                            peerConnection?.addIceCandidate(p0)
-                            Timber.d("On IceCandidate Added")
+                            Timber.d("Event-IceCandidate Generated")
+                            Timber.d("Event-IceCandidate ${calls[callId]?.getCallState()?.value}")
+                            if (calls[callId]?.getCallState()?.value != CallState.ACTIVE){
+                                peerConnection?.addIceCandidate(p0)
```

Comment:
check if peerConnection exist before printing the log

Reply:
Peer connection should always exit on line 1425 `peerConnection = Peer(`